content/resources/chef_client_config/_index.md

```diff
@@ -91,7 +91,7 @@ properties_list:
 - property: config_directory
   ruby_type: String
   required: false
-  default_value: "/etc/chef"
+  default_value: "`/etc/chef/` on *nix-like systems and `C:\chef\` on Windows"
```

Comment:
The netlify job doesn't like this line:

```
3:27:52 PM: Error: Error building site: "/opt/build/repo/content/resources/chef_client_config/_index.md:94:1": failed to unmarshal YAML: yaml: line 93: found unknown escape character
```
```suggestion
  default_value: "`/etc/chef/` on *nix-like systems and `C:\\chef\\` on Windows"
```

Reply:
Any reason these are necessary?

Reply:
The netlify job is failing on this line: https://app.netlify.com/sites/chef-web-docs/deploys/5f8629c245b8a700076dab6a